src/main/kotlin/com/cognifide/gradle/aem/instance/InstanceSync.kt

```diff
@@ -484,6 +485,8 @@ class InstanceSync(aem: AemExtension, instance: Instance) : InstanceHttpClient(a
         return scripts.asSequence().map { evalGroovyScript(it, data, verbose) }
     }
 
+    fun nodes(block: Nodes.() -> Unit) = Nodes(this).block()
```

Comment:
still I insist to have `InstanceRepository` here instead of `Nodes`

InstanceRepository with both nodes related methods and properties


```kotlin
class InstanceRepository {


// Nodes CRUD

fun getNode(): Any
fun findNode(): Any?
// ...

// Properties CRUD

fun getProperty(): Any
fun findProperty(): Any?
// ...

}
```